custom/enikshay/tests/test_bets_updates.py

```diff
@@ -0,0 +1,144 @@
+import json
+import uuid
+from django.test import TestCase, override_settings, RequestFactory
+from corehq.apps.domain.models import Domain
+from corehq.apps.users.models import WebUser
+from custom.enikshay.integrations.bets.views import update_voucher, update_incentive, get_case
+from corehq.util.test_utils import create_and_save_a_case
+
+
+@override_settings(TESTS_SHOULD_USE_SQL_BACKEND=True)
+class TestBetsUpdates(TestCase):
+    domain = 'enikshay-bets-updates'
+
+    @classmethod
+    def setUpClass(cls):
+        super(TestBetsUpdates, cls).setUpClass()
+        cls.domain_obj = Domain(name=cls.domain, is_active=True)
+        cls.domain_obj.save()
+        cls.web_user = WebUser.create(cls.domain, 'blah', 'password')
+
+    @classmethod
+    def tearDownClass(cls):
+        cls.domain_obj.delete()
+        cls.web_user.delete()
+        super(TestBetsUpdates, cls).tearDownClass()
+
+    def make_request(self, view, data):
+        factory = RequestFactory()
+        request = factory.post("/a/enikshay/bets/{}".format(view.__name__),
+                               data=json.dumps(data),
+                               content_type='application/json')
+        request.user = self.web_user.get_django_user()
+        return view(request, self.domain)
```

Comment:
(non-blocking)
I generally prefer using django [`TestClient`](https://docs.djangoproject.com/en/1.10/topics/testing/tools/#the-test-client) instead of creating the request and view separately, as it creates a closer to production scenario, since the request will be passed through middleware and all that.